contentcuration/contentcuration/viewsets/user.py

```diff
@@ -266,7 +267,30 @@ def create_from_changes(self, changes):
 
     def delete_from_changes(self, changes):
         return self._handle_relationship_changes(changes)
-
+    @action(detail=True, methods=['delete'])
+    def remove_self(self, request, pk=None):
+        """
+        Allows a user to remove themselves from a channel as a viewer.
+        """
+        user = self.get_object()
+        channel_id = request.query_params.get('channel_id', None)
+
+        if not channel_id:
+            return Response({'detail': 'channel_id is required.'}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Lets return `HttpResponseBadRequest` here. You can specify "channel_id is required." as the message